### PR TITLE
set up initial files #3

### PR DESCRIPTION
**Pull request by Hanan795**
- set up initial files

### CODE REVIEW

**Review by hshahwan — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules
+package-lock.json
+README.md
```
Comment (hshahwan) [inline .gitignore]: why did you add readme.md here?

**Review by hshahwan — CHANGES_REQUESTED** [review]
don't forget to create components folder

File: .gitignore (hunk 1 above)
Comment (Hanan795) [reply .gitignore]: I forget to delete it

**Review by hshahwan — APPROVED** [review]
